.readthedocs.yaml

```
@@ -0,0 +1,15 @@
+# Read the Docs configuration file for Sphinx projects
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the OS, Python version and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.12"
```

Can we use newer?
```suggestion
  os: ubuntu-24.04
  tools:
    python: "3.13"
```
Or latest?
```suggestion
  os: ubuntu-lts-latest
  tools:
    python: "3"
```